src/include/daos_prop.h

```diff
@@ -273,6 +273,17 @@ daos_prop_alloc(uint32_t entries_nr);
 void
 daos_prop_free(daos_prop_t *prop);
 
+/**
+ * Merge a set of new DAOS properties into a set of existing DAOS properties.
+ *
+ * \param[in]	old_prop	Existing set of properties
+ * \param[in]	new_prop	New properties - may override old entries
+ *
+ * \return	Newly allocated merged property
+ */
+daos_prop_t *
+daos_prop_merge(daos_prop_t* old_prop, daos_prop_t *new_prop);
```

Comment:
(style)  "foo* bar" should be "foo *bar"

Reply:
Ditto.

Reply:
Fixed this one, too.